meteor/client/ui/Settings/ConfigManifestSettings.tsx

```diff
@@ -794,17 +794,19 @@ export class ConfigManifestSettings<
 			case ConfigManifestEntryType.SELECT:
 			case ConfigManifestEntryType.LAYER_MAPPINGS:
 			case ConfigManifestEntryType.SOURCE_LAYERS:
-				return _.isArray(value) ? (
-					<React.Fragment>
-						<ul className="table-values-list">
-							{_.map((value as string[]) || [], (val) => (
-								<li key={val}>{val}</li>
-							))}
-						</ul>
-					</React.Fragment>
-				) : (
-					value.toString()
-				)
+				if (_.isArray(value)) {
+					return (
+						<React.Fragment>
+							<ul className="table-values-list">
+								{_.map((value as any[]) || [], (val) => {
```

Comment:
I'd suggest giving this a concrete type:
```suggestion
								{_.map((value as (string | { value: string })[]) || [], (val) => {
```

Reply:
Updated